optimum/habana/checkpoint_utils.py

```diff
@@ -58,7 +58,9 @@ def get_checkpoint_files(model_name_or_path, local_rank, token=None):
 
     # Extensions: .bin | .pt
     # Creates a list of paths from all downloaded files in cache dir
-    file_list = [str(entry) for entry in Path(cached_repo_dir).rglob("*.[bp][it][n]") if entry.is_file()]
+    path = Path(cached_repo_dir)
+    globs = ["*.bin", "*.pt", "*.safetensors"]
```

Comment:
Don't we need some kind of if else check ; cached repo dir can have both .safetensors and .pt files.

Reply:
The code that is already there loads `.bin` and `.pt` files, can the cached repo have both of those as well? I might be misunderstanding something

Reply:
Yes it can have both.